tests/jerry/es2015/symbol.js

```diff
@@ -120,3 +120,32 @@ var obj = { foobar : 55 };
 obj[symA] = 77;
 assert (obj["foobar"] !== obj[symA]);
 assert (obj["foobar"] != obj[symA]);
+
+try {
+  undefined[Symbol ("foo")];
+  assert (false);
+} catch (e) {
+  /* Error message: Cannot read property 'Symbol(foo)' of undefined */
+  assert (e instanceof TypeError);
+}
+
+var typedArrayTypes = [Int8Array,
```

Comment:
This test-case should be on it's own file as this requires the TypedArray and Symbol support.